### ISSUE TITLE
Update Babel to 7.x

### ISSUE THREAD

**Issue by dominik-zeglen** [survivejs/react-component-boilerplate#45]
I think it would be beneficial to update Babel to 7.x version - especially for `<>` tag support (`React.Fragment`) and keeping dependencies always up to date.

**Comment by bebraw** [comment]
Yeah, agreed. I would say the right time to update is when they go stable with it. Better leave this open till then.

**Comment by bebraw** [comment]
It's stable now. PR welcome (unless I get to it first).
